lib/game_team.rb

```diff
@@ -1,24 +1,24 @@
-class Gameteam 
+class GameTeam
+  attr_reader :game_id, :team_id, :hoa, :result, :settled_in,
+              :head_coach, :goals, :shots, :tackles, :pim,
+              :powerPlayOpportunities, :powerPlayGoals, :faceOffWinPercentage,
+              :giveaways, :takeaways
 
-  attr_reader :game_id, :team_id, :hoa, :result, :settled_in, :head_coach, 
-              :goals, :shots, :tackles, :pim, :powerPlayOpportunities, 
-              :powerPlayGoals, :faceOffWinPercentage, :giveaways, :takeaways
-
-  def initialize(game_id, team_id, hoa, result, settled_in, head_coach, goals, shots, tackles, pim, powerPlayOpportunities, powerPlayGoals, faceOffWinPercentage, giveaways, takeaways)
-    @game_id = game_id
-    @team_id = team_id
-    @hoa = hoa
-    @result = result
-    @settled_in = settled_in
-    @head_coach = head_coach
-    @goals= goals
-    @shots = shots
-    @tackles = tackles
-    @pim = pim
-    @powerPlayOpportunities = powerPlayOpportunities
-    @powerPlayGoals = powerPlayGoals
-    @faceOffWinPercentage = faceOffWinPercentage
-    @giveaways = giveaways
-    @takeaways = takeaways
+  def initialize(game_team)
+    @game_id = game_team[:game_id].to_i
```

Comment:
we had discussed leaving ids that were not going to have math done to them to be left as strings, I don't think it will actually affect anything but we should leave it consistent.